docs/RunningOnDeviceIOS.md

```diff
@@ -27,7 +27,7 @@ When you run your app on device, we pack all the JavaScript code and the images
 
 1. Open `AwesomeApp/ios/AwesomeApp/AppDelegate.m`
 2. Uncomment `jsCodeLocation = [[NSBundle mainBundle] ...`
-3. The JS bundle will be built for dev or prod depending on your app's scheme (Debug = development build with warnings, Release = minified prod build with perf optimizations).
+3. The JS bundle will be built for dev or prod depending on your app's scheme (Debug = development build with warnings, Release = minified prod build with perf optimizations). To change it navigate to `Product > Scheme > Edit Scheme...` and change `Build Configuration` between `Debug` and `Release`.
```

Comment:
you can be more clear here. Something like `To change the scheme navigate to ....... in Xcode and change ........`